.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "[deps] "
+    open-pull-requests-limit: 3
```

Comment:
Does this mean we can only have 3 open PRs at a time for good practice?

Reply:
I believe this means that dependabot will at max have 3 open PR for dependency updates.